classic.sh

```diff
@@ -24,33 +24,40 @@ case "$(uname -m)" in
         ;;
 esac
 
-if [ ! -f "xenial-base-$arch.tar.gz" ]; then
-    echo "Downloading xenial chroot for $arch..."
-    python3 -c "from urllib.request import urlretrieve; urlretrieve('http://cdimage.ubuntu.com/ubuntu-base/xenial/daily/current/xenial-base-$arch.tar.gz', 'xenial-base-$arch.tar.gz')"
+if [ ! -f "classic-base-$arch.tar.gz" ]; then
+    if [[ "$DEVEL" == "yes" ]]; then
+        echo "Downloading Ubuntu 16.10 (Development Branch) $arch Classic Base image..."
+        python3 -c "from urllib.request import urlretrieve; urlretrieve('http://cdimage.ubuntu.com/ubuntu-base/daily/current/yakkety-base-$arch.tar.gz', 'classic-base-$arch.tar.gz')"
```

Comment:
You could make so that it saves the file as xenial or yakkety and simply reuse the name below. This way the downloaded file would be unambiguous.